bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "finitio",
-  "version": "0.1.0",
+  "version": "0.2.1",
```

Comment:
Btw, I would actually go for 1.0.0. Finitio.js is used in production for a while and changing all dependencies at once might potentially hurt some projects. Let announce it clearly with a major version bump.